lib/src/utils.dart

```diff
@@ -145,8 +140,43 @@ extension NullableDurationRrule on Duration? {
       this == null || (0.days <= this! && this! <= 1.days);
 }
 
-extension IntRrule on int {
-  Duration get weeks => (this * 7).days;
+extension IntRange on int {
+  ///Copied from supercharged_dart
```

Comment:
Nit: Using a `///` adds this line to the doc comment of the next declaration, `rangeTo`.

```suggestion
  // Copied from supercharged_dart
```